src/style.css

```diff
@@ -0,0 +1,19 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.colored-blocks__red {
+  background-color: rgb(255, 0,0);
+  height: 100px;
+}
+
+.colored-blocks__green {
+  background-color: green;
+  height: 100px;
+}
+
+.colored-blocks__blue {
+  background-color: rgb(0, 0,255);
+  height: 100px;
+}
```

Comment:
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.